app/presenters/formats/edition_format_presenter.rb

```diff
@@ -14,17 +14,7 @@ def render_for_publishing_api(republish: false)
     attr_reader :edition, :artefact
 
     def optional_fields
-      fields = {}
-
-      if edition.auth_bypass_id
-        fields[:access_limited] = { auth_bypass_ids: [edition.auth_bypass_id] }
-      end
-
-      if !edition.live? && edition.phase_message
-        fields[:phase_message] = edition.phase_message
-      end
-
-      fields
+      { access_limited: access_limited }.delete_if { |_key, value| value.blank? }
```

Comment:
I'm finding this quite hard to understand. Could we use a different technique here?

```
{
   access_limited: access_limited,
   ...
}.compact
```